docs/allow-nested-if-parent-exist.md

```diff
@@ -0,0 +1,51 @@
+# AllowNestedIfParentExist
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-11
+
+## Description
+
+List calls for nested children under the resource group segment is allowed only if parent resource under the resource group exist.
+
+## How to fix
+
+Nested List call without parent call is invalid, ensure to add Parent Resource get call as below valid example or remove the Nested get call from the paths.
+
```

Comment:
To fix this, ensure that the LIST call for the child resource has a resource group segment